python/Ganga/Core/GangaRepository/SessionLock.py

```diff
@@ -424,15 +404,14 @@ def startup(self):
                 logger.debug("Startup Session Exception: %s" % err)
                 raise RepositoryError(self.repo, "Error on session file '%s' creation: %s" % (self.fn, err))
 
-            session_lock_refresher = getGlobalLockRef(self.session_name, self.sdir, self.gfn, self.afs)
+            local_session_lock_refresher = getGlobalLockRef(self.session_name, self.sdir, self.gfn, self.afs)
```

Comment:
This was to track the fact that this is a local variable effectively and doesn't confuse the local variable with the global one.
We could ignore the returned value from ```getGlobalLockRef``` and simply get the global object again but it's returned by this function so we might as well use it.
This helps make it clear where the object is coming from and that we're not reassigning the global object here. (there is no need for the global)